### ISSUE TITLE
Dateparser fails if local data package exists

### ISSUE THREAD

**Issue by mattwoods11** [scrapinghub/dateparser#352]
The current way dateparser configures its settings does not seem to play nice if you have a local data package in your project. 

Your operating system name and version: macOS Sierra 10.12.5
```
(venv)$ python --version
Python 3.5.1
(venv)$ pip freeze | grep dateparser
dateparser==0.6.0
```
Any details about your local setup that might be helpful in troubleshooting: I currently make use of a data package within my project for database related functions.
Detailed steps to reproduce the bug:

1) Create a data package within your project. My current structure looks something like this:
   project_name-->project_name-->data

2) Attempt to import dateparser anywhere in the project outside of the data package. This will cause the following Traceback:

```
Traceback (most recent call last):
  File "abc.py", line 1, in <module>
    import dateparser
  File "/PycharmProjects/test_dateparser/venv/lib/python3.5/site-packages/dateparser/__init__.py", line 4, in <module>
    from .date import DateDataParser
  File "/PycharmProjects/test_dateparser/venv/lib/python3.5/site-packages/dateparser/date.py", line 11, in <module>
    from dateparser.date_parser import date_parser
  File "/PycharmProjects/test_dateparser/venv/lib/python3.5/site-packages/dateparser/date_parser.py", line 10, in <module>
    from .conf import apply_settings
  File "/PycharmProjects/test_dateparser/venv/lib/python3.5/site-packages/dateparser/conf.py", line 66, in <module>
    settings = Settings()
  File "/PycharmProjects/test_dateparser/venv/lib/python3.5/site-packages/dateparser/conf.py", line 32, in __init__
    self._updateall(self._get_settings_from_yaml().items())
  File "/PycharmProjects/test_dateparser/venv/lib/python3.5/site-packages/dateparser/conf.py", line 45, in _get_settings_from_yaml
    data = get_data('data', 'settings.yaml')
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/pkgutil.py", line 630, in get_data
    return loader.get_data(resource_name)
  File "<frozen importlib._bootstrap_external>", line 815, in get_data
FileNotFoundError: [Errno 2] No such file or directory: '/PycharmProjects/test_dateparser/data/settings.yaml'
```

**Comment by akucukelbir** [comment]
I am having the same problem.

@mattwoods11 did you find a decent workaround to this?

**Comment by asadurski** [comment]
@mattwoods11 @akucukelbir: This problem should be resolved in version 0.7.0 (changed directory names). Could you confirm that?

**Comment by mattwoods11** [comment]
Version 0.7.0 appears to resolve this issue. Thank you!
